### PR TITLE
Add basic information

### PR DESCRIPTION
**Pull request by derjubu**
Add basic HTML structure an information for website.

### CODE REVIEW

**Comment by PReichetanz** [comment]
Nice work, I did not find anything to change here!

File: index.html [hunk 1]
```diff
@@ -22,60 +22,103 @@ <h1>Julian Buchholz</h1>
...
         <h2>Skills</h2>
         <dl>
+          <dt>Gif expertise</dt>
+          <dd aria-label="5 out of 5">
+            <span class="fa fa-star text-yellow-400"></span>
+            <span class="fa fa-star text-yellow-400"></span>
+            <span class="fa fa-star text-yellow-400"></span>
+            <span class="fa fa-star text-yellow-400"></span>
+            <span class="fa fa-star text-yellow-400"></span>
+          </dd>
```
Comment (badabam) [inline index.html:35]: Wow! ⭐ ⭐ ⭐ ⭐ ⭐

File: index.html [hunk 2]
```diff
@@ -22,60 +22,103 @@ <h1>Julian Buchholz</h1>
...
           <dt>CSS</dt>
-          <dd>
+          <dd aria-label="2 out of 5">
+            <span class="fa fa-star text-yellow-400"></span>
+            <span class="fa fa-star text-yellow-400"></span>
+            <span class="fa fa-star"></span>
+            <span class="fa fa-star"></span>
+            <span class="fa fa-star"></span>
+          </dd>
+          <dt>JacaScript</dt>
```
Comment (badabam) [inline index.html:68]: Little typo here.

File: index.html [hunk 3]
```diff
@@ -22,60 +22,103 @@ <h1>Julian Buchholz</h1>
...
-      </div>
-      <div>
-        <h2>Former employees</h2>
-      </div>
-      <div>
-        <h2>Contact</h2>
-      </div>
+      </section>
+      <section>
+        <div>
```
Comment (badabam) [inline index.html:87]: You don't need this div – you already have a section around this content

File: index.html [hunk 4]
```diff
@@ -22,60 +22,103 @@ <h1>Julian Buchholz</h1>
...
+          <h2>Showcase</h2>
+          View my work at
+          <a href="https://codepen.io/collection/KpdkKw">Codepen.</a>
+        </div>
+      </section>
+      <section>
+        <div>
```
Comment (badabam) [inline index.html:94]: Unnecessary div – please check everywhere.

**Review by badabam — COMMENTED** [review]
Very nice work! You could get rid of some divs though.

**Pull request merged**